services/src/main/java/org/keycloak/services/clienttype/impl/DefaultClientType.java

```diff
@@ -93,40 +87,9 @@ public void onCreate(ClientRepresentation createdClient) throws ClientTypeExcept
             if (propertyConfig.getDefaultValue() != null) {
```

Comment:
I'm a bit confused on what would be better to put here, any suggestions?

Reply:
@graziang @vickeybrown Sorry, I missed the context here...

I think that line `if (!propertyConfig.getApplicable()) continue;`  is OK. In case that some client property is not applicable for particular client type (EG. property `alwaysDisplayInConsole` is not applicable for client type `service account`), then we don't need to deal with this property and just ignore it IMO.

Reply:
My point is just that if you have `applicable = false`, for instance you can create a client with `alwaysDisplayInConsole = true` but the value returned in `TypeAwareClientModelDelegate` will be `false` using `getBooleanProperty()`. But as you said, the purpose of the `applicable = false` is to ignore the field and hide it from the admin console, so doesn't matter what values that field takes on, right?

Reply:
Yes, exactly. The field which is not `applicable` should be hidden from the admin console, so return value should not matter.